epictrack-api/src/api/services/project.py

```diff
@@ -186,14 +189,16 @@ def check_first_nation_available(cls, project_id: int, work_id: int) -> bool:
         return {"first_nation_available": result}
 
     @classmethod
-    def import_projects(cls, file: IO):
+    def import_projects(cls, file: IO):  # pylint: disable=too-many-locals

```

Comment:
this function is too big, can we break it into different small functions with proper naming

Reply:
Updated